README.md

```diff
@@ -53,6 +53,35 @@ Add.call # => [:ok, 42]
 Add.call(a: 660, b: 6) # => [:ok, 666]
 ```
 
+## Default Values
+
+```ruby
+# You can define defaults via key/value pairs, as so:
+class Add
+  include Verbalize::Action
+  # note that these values are evaluated at load-time as they are not wrapped
+  # in lambdas.
+  input optional: [a: 35, b: 7]
```

Comment:
I missed the review period on this PR, but I have discussed this before with @taylorzr and I pretty strongly disagree with providing functionality which allows users to define mutable default values.  Obviously this is not an issue for the example with integers, but most value types are mutable in ruby, so this requires the user to use extreme caution when writing Actions with defaults. It would be strongly preferable IMO to simply not support this and/or implement an inputs/defaults syntax that is not susceptible to this issue.